test/integration-tests/longCallSpreadExpireItm.test.ts

```diff
@@ -339,22 +339,22 @@ contract('Long Call Spread Option expires Itm flow', ([accountOwner1, nakedBuyer
 
       // Check that we end at a valid state
       const vaultAfter = await controllerProxy.getVault(accountOwner1, vaultCounter1)
-      const vaultStateAfter = await calculator.getExcessCollateral(vaultAfter)
+      const vaultStateAfter = await calculator.getExcessCollateral(vaultAfter[0], vaultAfter[1])
       assert.equal(vaultStateAfter[0].toString(), '0')
       assert.equal(vaultStateAfter[1], true)
```

Comment:
similarly, I think it's better to use 
```javascript
const [vaultAfter, ] = await controllerProxy.getVault(accountOwner1, vaultCounter1)
```

so u don't have to use `vaultAfter[0]`